### DISCUSSION TITLE
panDirection is saved with custom layouts.

### DISCUSSION THREAD

**Discussion by Peatote** [f1multiviewer/issue-tracker#131] [Feature requests] [answered]
It seems like the preferred panDirection setting to hear less of the engine and more of the team radio is the same for that driver's OBC in each race. I've only tested this out with a few different drivers on a couple races, though, so I could be mistaken.

If it typically is the same, It'd be great to have the panDirection setting for a driver's OBC saved with custom layouts. 

Thank you!

**Comment by f1multiviewer** [comment, marked as answer]
It does seem pretty stable in general. I think what I'll do, is try and go over each driver and use defaults like this. So you can just enable "engine noise suppression" mode or something (when I add the UI)

**Reply by Peatote** [reply]
Ooh that sounds better! I like it. Thanks, again
